tests/_internal/concurrency/test_services.py

```diff
@@ -308,3 +309,22 @@ class IntervalMockBatchedService(MockBatchedService):
     IntervalMockBatchedService.mock.assert_has_calls(
         [call(instance, [1]), call(instance, [2])]
     )
+
+
+def test_context_available_to_handler():
+    class ContextAwareService(QueueService[int]):
+        mock = MagicMock()
+
+        async def _handle(self, item):
+            context = copy_context()
+            for key, value in context.items():
+                if key.name == "the-answer":
+                    self.mock(value)
+
+    ContextAwareService.instance().send(1)
+
+    ContextVar("the-answer").set(42)
```

Comment:
You're not supposed to declare context vars outside the top-level scope for some reason — I'd declare a constant at the top of the test file.